packages/flutter/lib/src/widgets/spell_check.dart

```diff
@@ -0,0 +1,53 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/painting.dart';
+import 'package:flutter/services.dart'
+    show SpellCheckResults, SpellCheckService, TextEditingValue;
+
+/// Controls how spell check is performed for text input.
+///
+/// This configuration determines the [SpellCheckService] used to fetch the
+/// [List<SuggestionSpan>] spell check results and the
+/// [SpellCheckSuggestionsHandler] used to mark and display replacement
+/// suggestions for misspelled words within text input.
+class SpellCheckConfiguration {
+  /// Creates a configuration that specifies the service and suggestions handler
+  /// for spell check.
+  SpellCheckConfiguration(
+    {this.spellCheckService, this.spellCheckSuggestionsHandler}
```

Comment:
Nit: I think usually the curly braces usually go on the same lines as the parentheses, but not sure if that's required here.